src/panel/history/HistoryPanel.jsx

```diff
@@ -15,6 +46,147 @@ const HistoryPanel = () => {
     setHistoryEnabled(e.target.checked);
   };
 
+  const visit = item => {
+    // PoI
+    if (item.type === 'poi') {
+      window.app.navigateTo(`/place/${item.item.id}`);
+    }
+    // Intention
+    else if (item.type === 'intention') {
+      // With category
+      if (item.item.category && item.item.category.name) {
+        window.app.navigateTo(
+          `/places/?type=${item.item.category.name}&bbox=${item.item.bbox.join(',')}`
+        );
+      }
+      // Without category (ex: macdonalds nice)
+      else {
+        window.app.navigateTo(
+          `/places/?q=${item.item.fullTextQuery}&bbox=${item.item.bbox.join(',')}`
+        );
+      }
+    }
+  };
+
+  // Remove one item from the list
+  const remove = item => {
+    // Remove the item in localStorage
+    deleteQuery(item);
+
+    // Refresh local copy of localStorage to re-render the page
+    setHistory(getHistory());
+  };
+
+  // Clear all the history
+  const clear = () => {
+    // Temp: a nice popup will appear here soon
+    if (confirm('Are you sure?')) {
+      // Clear history in localStorage
+      deleteSearchHistory();
+      // Refresh local copy of localStorage to re-render the page
+      setHistory(getHistory());
+    }
+  };
+
+  // TEMP: fill history with items (poi, address, city, intention) from different dates (< 24h, < 1 week, < 2 weeks, < 3 weeks, older)
+  // prettier-ignore
+  const demo = () => {
```

Comment:
it could be moved into a test once you have done. This comment is also to not forget to remove it